backends/azure/azure.go

```diff
@@ -0,0 +1,495 @@
+package azure
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"sort"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+	"github.com/PowerDNS/go-tlsconfig"
+	"github.com/PowerDNS/simpleblob"
+	"github.com/go-logr/logr"
+	"github.com/sirupsen/logrus"
+)
+
+// Azure blob implementation examples can be found here:
+//https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/storage/azblob/examples_test.go
+
+const (
+	// DefaultEndpointURL is the default the local Azurite default endpoint
+	DefaultEndpointURL = "http://127.0.0.1:10000/devstoreaccount1"
```

Comment:
Having a development URL here doesn't feel right, could you make it more similar as the way the S3 backend is configured please?